ge/org.osate.ge/src/org/osate/ge/ContentFilter.java

```diff
@@ -24,25 +24,45 @@
 package org.osate.ge;
 
 /**
- * A configurable content filters is one whose enablement can be toggled by the user. Must be stateless.
+ * A configurable content filters defines a set of business object which can be optionally shown on a diagram. Must be stateless.
```

Comment:
Change filters to filter.  Change object to objects

Reply:
Should be fixed